test/test_J.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import pytest
+import time
+
+import maxwell.dg.dg1d_tools as dg
+import maxwell.dg.mesh1d as ms
+from maxwell.driver import *
+from maxwell.dg.mesh1d import *
+from maxwell.dg.dg1d import *
+from maxwell.fd.fd1d import *
+
+
+from nodepy import runge_kutta_method as rk
+
```

Comment:
Is it not enough with the _real_ tests? We could look at those for documentation